promisasu/DMTBPortal
# Sprint6

**Taiga User Story** : 132 and 116

**Taiga Task(s)** : For user story 132  tasks are : 137,138,139  and 
 for story 116 tasks are 121,122,127,129,131,128,,130

**What did you change?** Query isolation, code refactoring and regression testing.

**How can we test?**
- clone the Sprint6
- run ant startportal from root directory of project.
- check portal functionality on localhost:3000
